azure-pipelines/integrationtests.yml

```diff
@@ -0,0 +1,52 @@
+pool:
+    name: SqlToolsTestAgentPool
+  steps:
+  - task: DownloadSecureFile@1
+    displayName: 'Download secure file'
+    inputs:
+      secureFile: 'aec0999c-ce6e-49b4-a386-edd5729f1b0f'
+  
+  - task: VisualStudioTestPlatformInstaller@1
+    displayName: 'Visual Studio Test Platform Installer'
+    inputs:
+      versionSelector: latestStable
+  
+  - task: UseDotNet@2
+    displayName: 'Use .NET Core sdk '
+    inputs:
+      useGlobalJson: true
+  
+  - task: DotNetCoreCLI@2
+    displayName: 'dotnet build'
+    inputs:
+      projects: '**/Microsoft.SqlTools.ServiceLayer.IntegrationTests.csproj'
+  
+  - task: CopyFiles@2
+    displayName: 'Copy testsettings file to bin'
+    inputs:
+      SourceFolder: '$(Agent.TempDirectory)'
+      Contents: '**\testsettings.json'
+      TargetFolder: '$(Build.SourcesDirectory)\test\Microsoft.SqlTools.ServiceLayer.IntegrationTests\bin\Debug\netcoreapp3.1'
+  
+  - task: DotNetCoreCLI@2
+    displayName: 'Run integration tests'
+    inputs:
+      command: test
+      projects: '**/Microsoft.SqlTools.ServiceLayer.IntegrationTests.csproj'
+      arguments: '--no-build'
+      testRunTitle: 'SqlToolsService Integration Tests'
+    enabled: false
+  
+  - task: VSTest@2
+    displayName: 'Run integration tests with code coverage'
+    inputs:
+      testAssemblyVer2: '**/bin/Debug/netcoreapp3.1/Microsoft.SqlTools.ServiceLayer.IntegrationTests.dll'

```

Comment:
`**/bin/Debug/**/Microsoft.SqlTools.ServiceLayer.IntegrationTests.dll`? One less thing to update if we update the netcore version